.github/config.yml

```diff
@@ -0,0 +1,44 @@
+#-------------------------------------------------------------------------------
+# Configuration for new-issue-welcome - https://github.com/behaviorbot/new-issue-welcome
+ # Comment to be posted to on first time issues
+newIssueWelcomeComment: >
+   Thanks for opening this issue. A contributor will be by to give feedback soon. In the meantime, please review the [Layer5 Contributors' Welcome Guide](https://docs.google.com/document/d/17OPtDE_rdnPQxmk2Kauhm3GwXF1R5dZ3Cj8qZLKdo5E/edit?usp=sharing) and sure to join the [community Slack](http://slack.layer5.io/).
+ # Configuration for new-pr-welcome - https://github.com/behaviorbot/new-pr-welcome
+ # Comment to be posted to on PRs from first time contributors in your repository
+newPRWelcomeComment: >
+   Yay, your first pull request! :thumbsup: A contributor will be by to give feedback soon. In the meantime, please review the [Layer5 Contributors' Welcome Guide](https://docs.google.com/document/d/17OPtDE_rdnPQxmk2Kauhm3GwXF1R5dZ3Cj8qZLKdo5E/edit?usp=sharing) and sure to join the [community Slack](http://slack.layer5.io/).
+
+   Be sure to double-check that you have signed your commits. Here are instructions for [making signing an implicit activity while peforming a commit](https://github.com/layer5io/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin).
+
+
+#-------------------------------------------------------------------------------
+# Configuration for first-pr-merge - https://github.com/behaviorbot/first-pr-merge
+# Comment to be posted to on pull requests merged by a first time user
+firstPRMergeComment: >
+  Thanks for your contribution to the Layer5 community! :tada:
+   
+  ![Congrats!](https://raw.githubusercontent.com/layer5io/meshery/master/.github/welcome/Layer5-celebration.png)
+  
+                          :star: Please [star the project](../stargazers) if you have yet to do so.
```

Comment:
```suggestion
                          :star: Please [star the project](../stargazers) if you have yet to do so and sure to join the [community Slack](http://slack.layer5.io/).
```

Reply:
If I am not mistaken I think that maybe you meant to write "and BE sure to join.. " instead of "and sure to join.. ". Maybe I can correct it before making a  new pull request for the changes you pointed out.